portal-frontend/src/app/features/view-application/view-application.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Subject, takeUntil } from 'rxjs';
+import { ApplicationDto } from '../../services/application/application.dto';
+import { ApplicationService } from '../../services/application/application.service';
+import { ConfirmationDialogService } from '../../shared/confirmation-dialog/confirmation-dialog.service';
+
+@Component({
+  selector: 'app-view-application',
+  templateUrl: './view-application.component.html',
+  styleUrls: ['./view-application.component.scss'],
+})
+export class ViewApplicationComponent implements OnInit, OnDestroy {
+  application: ApplicationDto | undefined;
+
+  $destroy = new Subject<void>();
+
+  constructor(
+    private applicationService: ApplicationService,
+    private route: ActivatedRoute,
+    private confirmationDialogService: ConfirmationDialogService
+  ) {}
+
+  ngOnInit(): void {
+    this.route.paramMap.pipe(takeUntil(this.$destroy)).subscribe((routeParams) => {
```

Comment:
I wish destroy would be a default option, so we do not need to wright it every single time

Reply:
100%, not sure why there isnt an easier way